### PR TITLE
Move Interop.GetTimestamp.cs to shared CoreLib partition.

### PR DESCRIPTION
**Pull request by filipnavara**
Related to https://github.com/dotnet/corert/pull/6932.

### CODE REVIEW

**Comment by jkotas** [comment]
It may be nice to split this file into two. One file in shared that has just the timestamp APIs and second file in corefx only that has the other two APIs.

File: src/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj [hunk 1]
```diff
@@ -412,8 +412,8 @@
     <Compile Include="$(CommonPath)\Interop\FreeBSD\Interop.Process.cs">
       <Link>Common\Interop\FreeBSD\Interop.Process.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Interop\Unix\System.Native\Interop.GetTimestamp.cs">
-      <Link>Common\Unix\System.Native\Interop.GetTimestamp.cs</Link>
+    <Compile Include="$(CommonPath)\Interop\Unix\System.Native\Interop.GetAbsoluteTime.cs">
```
Comment (jkotas) [inline src/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj]: Is it needed here? It seems to be used by `Process.OSX.cs` only.

Comment (filipnavara) [reply src/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj]: Right, probably not.

**Comment by jkotas** [comment]
@dotnet-bot test OSX x64 Debug Build
@dotnet-bot test Packaging All Configurations x64 Debug Build

**Comment by jkotas** [comment]
@dotnet-bot test Packaging All Configurations x64 Debug Build

**Pull request merged**